specs/core/0_beacon-chain.md

```diff
@@ -878,61 +855,39 @@ def get_seed(state: BeaconState, epoch: Epoch, domain_type: DomainType) -> Hash:
     return hash(domain_type + int_to_bytes(epoch, length=8) + mix)
 ```
 
-#### `get_committee_count`
+#### `get_committee_count_at_slot`
 
 ```python
-def get_committee_count(state: BeaconState, epoch: Epoch) -> uint64:
+def get_committee_count_at_slot(state: BeaconState, slot: Slot) -> uint64:
     """
-    Return the number of committees at ``epoch``.
+    Return the number of committees at ``slot``.
     """
-    committees_per_slot = max(1, min(
-        SHARD_COUNT // SLOTS_PER_EPOCH,
+    epoch = compute_epoch_at_slot(slot)
+    return max(1, min(
```

Comment:
don't need `max(1,` since `MAX_COMMITTEES_PER_SLOT` will always be >1

Reply:
but the min operation could result in `0`

Say active validator indices is 10, then it looks like 

```
max(1, min(64, 10 // 64 // 128))
```

`10 // 64 // 128 == 0` so

```
max(1, min(64, 0)) == max(1, 0)
```

Reply:
Ahh I see. That's right